test/e2e/kubectl/kubectl.go

```diff
@@ -1993,21 +1994,6 @@ type updateDemoData struct {
 
 const applyTestLabel = "kubectl.kubernetes.io/apply-test"
 
-func readBytesFromFile(filename string) []byte {
```

Comment:
it's always nice to add a sentence or two in the PR description about removing unused code.

Reply:
Done